spec/models/ticket_spec.rb

```diff
@@ -82,17 +82,37 @@
     end
   end
 
+  describe '#unpaid?' do
+    let!(:ticket_purchase) { create(:ticket_purchase, user: user, ticket: ticket) }
+
+    context 'user has not paid' do
+      before { ticket_purchase.update_attributes(paid: false) }
+
+      it 'returns true' do
+        expect(ticket.unpaid?(user)).to eq(true)
+      end
+    end
+
+    context 'user has paid' do
+      before { ticket_purchase.update_attributes(paid: true) }
+
+      it 'returns false' do
+        expect(ticket.unpaid?(user)).to eq(false)
+      end
+    end
+  end
+
   describe '#quantity_bought_by' do
```

Comment:
@rishabhs95, if a method accepts a parameter, it must be tested with that parameter, regardless whether it has a default. So you have to test both cases, when `paid` is `true` and when it's `false`.